azure/config.go

```diff
@@ -46,7 +57,7 @@ func init() {
 	kindfn := func(u *url.URL) bool {
 		return u.Scheme == "azure"
```

Comment:
We have a const called `Kind` we would probably want to use here rather than the hard coded value.